x-pack/test/functional/services/ml/job_wizard_common.ts

```diff
@@ -510,5 +541,9 @@ export function MachineLearningJobWizardCommonProvider(
       await testSubjects.clickWhenNotDisabled('mlJobWizardButtonCreateJob');
       await testSubjects.existOrFail('mlJobWizardButtonRunInRealTime', { timeout: 2 * 60 * 1000 });
     },
+
+    async createJobAndDoNotWaitForCompletion() {
+      await testSubjects.clickWhenNotDisabled('mlJobWizardButtonCreateJob');
```

Comment:
We should validate that the click actually did something. In this case I'd suggest to add
```
await testSubjects.existOrFail('mlPageJobManagement');
```
And maybe a renaming of the service method could help to prevent confusion as it currently looks like it would be the same as `createJobAndWaitForCompletion` just without the waiting part - but actually it's navigating back to the job management. So maybe something like `createJobWithoutDatafeedStart` would be clearer?

Reply:
Updated in https://github.com/elastic/kibana/pull/77484/commits/252c1c7d5717f99efd4ce8c093e589d4fc6c0dbb